frontend/components/dashboard/creation/DashboardCreationController.vue

```diff
@@ -78,7 +78,7 @@ function show(
   }
   network.value = forcedNetwork || currentNetwork.value
   state.value = 'type'
-  name.value = isLoggedIn.value ? '' : 'cookie'
+  name.value = isLoggedIn.value ? '' : ''
```

Comment:
I am not entirely sure on my own what this `line` does...which means that the whole `feature` needs refactoring...but plz never do things like this, because this line no is `max-confusing`

```suggestion
```
(would exactly be the same)